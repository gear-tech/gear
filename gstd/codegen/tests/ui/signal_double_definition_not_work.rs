#![no_main]

#[gstd::async_init(handle_signal = custom_handle_signal)]
async fn init() {}

#[gstd::async_main(handle_signal = custom_handle_signal)]
async fn main() {}

fn custom_handle_signal() {}
