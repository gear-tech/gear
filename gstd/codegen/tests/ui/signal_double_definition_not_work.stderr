error: parameter already defined
 --> tests/ui/signal_double_definition_not_work.rs:6:36
  |
6 | #[gstd::async_main(handle_signal = custom_handle_signal)]
  |                                    ^^^^^^^^^^^^^^^^^^^^
